components/service/glean/scripts/sdk_generator.gradle

```diff
@@ -161,6 +161,7 @@ ext.gleanGenerateMetricsAPI = {
         args "-s"
         args "namespace=$fullNamespace"
         args "$projectDir/metrics.yaml"
+        args "$projectDir/pings.yaml"
```

Comment:
We pass both of these files to `glean_parser`, even though only one or the other might exist.  `glean_parser` is smart enough not to care if one of them is missing.  (Though I guess it would be weird to have `pings` without `metrics`, having `metrics` without `pings` is completely legit.)